modules/coreos-layering-configuring.adoc

```diff
@@ -32,8 +32,8 @@ For example, the following Containerfile creates a custom layered image from an
 # Using a {product-version}.0 image
 FROM quay.io/openshift-release/ocp-release@sha256... <1>
 #Install hotfix rpm
-RUN rpm-ostree override replace http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/kernel-{,core-,modules-,modules-core-,modules-extra-}5.14.0-295.el9.x86_64.rpm && \ <2>
-    rpm-ostree cleanup -m && \
+RUN dnf swap kernel http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/kernel-{,core-,modules-,modules-core-,modules-extra-}5.14.0-295.el9.x86_64.rpm && \ <2>
```

Comment:
OK, confusingly the kernel is special and currently still requires `rpm-ostree override replace` for $reasons.